Gemfile.lock

```diff
@@ -622,8 +622,8 @@ GEM
       sprockets (>= 2.8, < 4.0)
     state_machine (1.2.0)
     stringex (1.5.1)
-    stripe (5.11.0)
-    temple (0.8.2)
```

Comment:
What's this `temple` gem? Looks like it's used by HAML. I can still see it in my version of 3-0-stable... was it removed in master?

Reply:
Ok, it's not in `master`... :heavy_check_mark:

Reply:
it's a dependency of haml 5 (rails 4) but of haml 4 (rails 3)